ga4gh/backend.py

```diff
@@ -531,6 +539,96 @@ def callSetsGenerator(self, request):
                 return self._noObjectGenerator()
             return self._singleObjectGenerator(callSet)
 
+    def rnaQuantificationGenerator(self, request):
+        """
+        Returns a generator over the (rnaQuantification, nextPageToken) pairs
+        defined by the specified request.
+        """
+        if request.rnaQuantificationId is None:
+            #TODO; remove hard coded dataset ID
+            dataset = self.getDataset('MWtnLXAzLXN1YnNldA==')
+            return self._objectListGenerator(request,
+                                             dataset.getRnaQuantifications())
+        else:
+            compoundId = datamodel.RnaQuantificationCompoundId.parse(request.rnaQuantificationId)
+            rnaQuantificationName = compoundId.rnaQuantification
+            dataset = self.getDataset(compoundId.datasetId)
+            try:
+                rnaQuant = dataset.getRnaQuantification(rnaQuantificationName)
+            except exceptions.RnaQuantificationNotFoundException:
+                return self._noObjectGenerator()
+            return self._singleObjectGenerator(rnaQuant)
+
+    def expressionLevelGenerator(self, request):
```

Comment:
these are pretty long methods you're adding here... generally it's a good idea to keep methods to as few lines as possible so maintainers don't need to keep track of a lot of in-scope variables... for instance, in this method at least building the ExpressionLevel object could be moved to its own private method